vm/src/vm/runners/builtin_runner/output.rs

```diff
@@ -20,11 +24,31 @@ impl OutputBuiltinRunner {
     pub fn new(included: bool) -> OutputBuiltinRunner {
         OutputBuiltinRunner {
             base: 0,
+            pages: HashMap::default(),
+            attributes: HashMap::default(),
             stop_ptr: None,
             included,
         }
     }
 
+    pub fn from_segment(segment: &Relocatable, included: bool) -> Self {
+        Self {
+            base: segment.segment_index as usize,
+            pages: HashMap::default(),
+            attributes: HashMap::default(),
+            stop_ptr: None,
+            included,
+        }
+    }
```

Comment:
Why is this method needed?

Reply:
As mentioned in my other reply, the first thing the bootloader does is create a new output builtin to capture the output of the simple bootloader program. To do this, it allocates a new segment and uses it as the base. It's basically a `new` method with the `base` provided as argument.

Reply:
Shouldn't that be done by calling:
```
let mut output = OutputBuiltinRunner::new(true);
output.initialize_segments();
```
I don't see the need for this specific method

Reply:
The way the bootloader works, this method makes more sense (the hints always create a segment, then the builtin from it), so this felt natural. But it can be done the way you suggest.

Reply:
Dropped the method.